_includes/tutorials/finding-distinct/ksql/code/docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
       KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
       KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_LOG_RETENTION_MS: -1
```

Comment:
Just for my own education: is it custom to set infinite retention in tutorials?

Reply:
No, there is more discussion about how to properly update the tutorials to address the underlying change in behavior in https://github.com/confluentinc/kafka-tutorials/issues/695. The branch targeted by this PR (ksqldb-latest) is used for ksql release validation only, and not live on the tutorials site, which is why we are opting for quicker fixes in the meantime.